### DISCUSSION TITLE
Image on Ingredient

### DISCUSSION THREAD

**Discussion by FlorinBuffet** [TandoorRecipes/recipes#2507] [Q&A]
Hi there
When i create a recipe and have some ingredients, when you look at the ingredients afterwards you can see the image of the recipe. Is this intended like that? What happens if i have 5 Recipes to use that ingredient?
Best Regards
Florin

**Comment by vabene1111** [comment]
it randomly chooses a recipe for the food. In the future its planned to add actual images for foods but thats not done yet
